pkg/openstack/loadbalancer_test.go

```diff
@@ -593,3 +594,195 @@ func Test_getIntFromServiceAnnotation(t *testing.T) {
 		})
 	}
 }
+
+func Test_buildPoolCreateOpt(t *testing.T) {
+	type args struct {
+		protocol string
+		svcConf  *serviceConfig
+		service  *corev1.Service
+		lbaasV2  *LbaasV2
+	}
+	tests := []struct {
+		name string
+		args args
+		want pools.CreateOpts
+	}{
+		{
+			name: "test for proxy protocol enabled",
+			args: args{
+				protocol: "TCP",
+				svcConf: &serviceConfig{
+					keepClientIP:                true,
+					tlsContainerRef:             "tls-container-ref",
+					healthMonitorDelay:          1,
+					healthMonitorTimeout:        1,
+					healthMonitorMaxRetries:     1,
+					healthMonitorMaxRetriesDown: 1,
+					healthCheckNodePort:         2,
```

Comment:
I think you don't need to specify these, the function doesn't seem to use them? Same goes for other cases.